web/static/sass/screen.scss

```diff
@@ -295,8 +302,16 @@ code {
     margin-top: 80px;
 
     @media #{$large-screen} {
-      position: absolute;
       bottom: 0;
+      position: absolute;
+    }
+
+    @media #{$not-tall-screen},
+           #{$not-large-screen},
+           #{$large-screen} and #{$not-tall-screen} {
+
+      margin: 0;
```

Comment:
What if we gave it some top margin so that it stayed separated from the other items?